Gems/LmbrCentral/Code/Source/Shape/EditorBaseShapeComponent.cpp

```diff
@@ -172,11 +172,13 @@ namespace LmbrCentral
 
     void EditorBaseShapeComponent::OnShapeColorChanged()
     {
-        m_shapeColor.SetA(AzFramework::ViewportColors::DeselectedColor.GetA());
+        // m_shapeColor.SetA(AzFramework::ViewportColors::DeselectedColor.GetA());
```

Comment:
nit: Can we delete this commented line please (also could add a brief comment to explain what this new logic is needed for)